start.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker run -it --rm --volume `PWD`/work:/home/jovyan/work/ superml /bin/bash
```

Comment:
I've fixed it like 
`docker run -it --rm -p 8888:8888 --volume $(pwd)/work:/home/jovyan/work/ superml /bin/bash`
and I checked that it works.

Reply:
soory